### DISCUSSION TITLE
How to import archived data (gz) from 'Request your archive'?

### DISCUSSION THREAD

**Discussion by kekyo** [mastodon/mastodon#28832] [Q&A]
I would like to migrate my existing mastodon account to another new mastodon instance account.
I want to migrate all toots and images as well.

So I exported and saved a gz file from 'request your archive' on my existing mastodon account.
How do I import this in the new instance?

**Comment by StrangeMinds1960** [comment]
From Settings > Import and export > Import....but it only looks like you can only import various lists, not posts or media

**Comment by kekyo** [comment]
Thanks your response! Oh, that's such a shame... I would worry less about maintaining my account if I could import it...

If we have any ideas for other solutions, I will continue to wait. For example, we can do it in a way that is possible if you are the server administrator by 3rd-party tools.
